dm/ctl/ctl.go

```diff
@@ -100,6 +101,13 @@ func HasCommand(name string) bool {
 	return false
 }
 
+// PrintHelp print help message for special subCommand
+func PrintHelp(args []string) {
+	cmd, _, _ := rootCmd.Find(args)
```

Comment:
I think if we reach here, it means we must have subcommand, because of this [condition](https://github.com/pingcap/dm/pull/287/files#diff-61075abbd9aac2de49a1fe702632b302R127), so we should always find subcommand here

Reply:
or when we got an error, how about printing rootCmd usage here?

Reply:
> or when we got an error, how about printing rootCmd usage here?

OK